stbt_auto_selftest.py

```diff
@@ -77,7 +80,6 @@ def main(argv):
     subparsers.add_parser('validate', help='Run (and check) the auto-selftests')
 
     cmdline_args = parser.parse_args(argv[1:])
-    screenshots_root = cmdline_args.screenshots_root or "selftest/screenshots"
```

Comment:
What if you run `stbt auto-selftests` outside the test-pack directory (e.g. in your CI's temporary directory, or in your stbt results directory)? Is the `--screenshots-root` still relative to the working directory? I suppose that some people might want to keep their screenshots out of the git repo?

Note that you won't be able to change the location (within your test-pack) and still give a relative path to it. I suppose that's ok, we don't need to support that.

Either way, we should make sure that giving an absolute path always does the right thing.